ocaml/testsuite/tests/typing-local/local.ml

```diff
@@ -2648,7 +2648,7 @@ Lines 3-6, characters 6-3:
 Error: Signature mismatch:
        Modules do not match:
          sig
-           val g : 'a -> 'b -> local_ string
+           val g : 'a -> ('b -> local_ string)
```

Comment:
The extra parentheses is because the first argument of `g` is unconstrained, and thus translates to `Olm_unknown` in `oprint.ml`. "unknown" is not equal to anything including another "unknown", causing the extra parentheses. And finally, we don't print the "unknown" around the parentheses. 

This is caused by our changes in `oprint.ml` of course, which I believe is slightly more correct?

Reply:
OK I think I know how to "fix" this.